### PR TITLE
feat: [IA-694] Add webview to show and communicate with donation web page

### PR DESCRIPTION
**Pull request by Undermaken**
## Short description
This PR adds a screen that embeds a webview to show and communicate with donation web page

## List of changes proposed in this pull request
- add `UAWebViewScreen.tsx` screen
- add navigator
- update `ALLOWED_ROUTE_NAMES`

### happy case
https://user-images.githubusercontent.com/822471/157063617-75ca627a-0240-48df-9761-458cf8f148f9.mov

### error case
(to reproduce turn off your internet connection)

https://user-images.githubusercontent.com/822471/157063767-fe174351-1108-4ea7-aaee-f4bffdb0e660.mov

## How to test
- turn ON the FF
- embed this code in a component where you can access to dispatch object

```typescript
 handleInternalLink(
      dispatch,
      "ioit://UADONATION_ROUTES_WEBVIEW?urlToLoad=https://www.google.it"
    );
```

### CODE REVIEW

**Review by thisisjp — APPROVED** [review]

File: ts/features/uaDonations/navigation/navigator.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    }
+  },
+  {
+    // Let each screen handle the header and navigation
+    headerMode: "none",
+    defaultNavigationOptions: {
+      gesturesEnabled: false
+    }
+  }
+);
```
Comment (fabriziofff) [inline ts/features/uaDonations/navigation/navigator.ts:18]: In order to be ready for the upcoming update with `react-navigation v5`, if no other screens are expected in this navigation flow, could we avoid to add a stack navigator and add direcly the 
```
    [UADONATION_ROUTES.WEBVIEW]: {
      screen: UAWebViewScreen
    }
```

in the `AppNavigator`?
If so, could you add also:

`[UADONATION_ROUTES.WEBVIEW] : undefined;`

in https://github.com/pagopa/io-app/blob/master/ts/navigation/params/AppParamsList.ts#L14-L13 ?

**Review by fabriziofff — CHANGES_REQUESTED** [review]
I didn't reviewed the whole pull request, just from the `react-navigation v5` point of view.

File: ts/features/uaDonations/navigation/navigator.ts (hunk 1 above)
Comment (Undermaken) [reply ts/features/uaDonations/navigation/navigator.ts:18]: sure! 
I hope I hit the point
https://github.com/pagopa/io-app/pull/3813/commits/c877e0f81968a2dd35220328a214fe3ba9d3c2d9

Comment (fabriziofff) [reply ts/features/uaDonations/navigation/navigator.ts:18]: This for the paramslist, but if no other screens are expected, we should also remove the UADonationNavigator and use direcly:

```
  [UADONATION_ROUTES.WEBVIEW]: {
      screen: UAWebViewScreen
    }
```

in `AppNavigator`.

**Review by emiliopavia — APPROVED** [review]
It looks fine to me. I also tested some edge cases (e.g. non-existent website) and it handles them correctly.

**Comment by pietro909** [comment]
Tested locally with https://github.com/pagopa/io-dev-api-server/pull/145 and works fine 🚀

File: ts/features/uaDonations/navigation/navigator.ts (hunk 1 above)
Comment (Undermaken) [reply ts/features/uaDonations/navigation/navigator.ts:18]: Thank for your patience
addressed here https://github.com/pagopa/io-app/pull/3813/commits/93fd9c7875758b1768acf468c59eb76d696d0fc0

**Review by fabriziofff — APPROVED** [review]
